<launch>
	<node name="autopilot_interface_node" pkg="autopilot_interface" type="autopilot_interface_node" output="screen">
		<!--param name="watchdog_binary" value="/usr/local/bin/uavAP/Watchdog" /-->
		<param name="serial_port" value="/dev/ttyUSB2" />
		<param name="watchdog_config" value="/usr/local/config/watchdog.json" />
		<param name="alvolo_config" value="/usr/local/config/alvolo.json" />
		<param name="log_file" value="/home/mircot/Documents/flights/pulsar/may_7_2018/DAQ_351.log" />
		<param name="log_header" value="/home/mircot/Documents/flights/pulsar/may_7_2018/351header.txt" />
	</node>
</launch>
